src/index.html

```diff
@@ -11,7 +11,110 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="" onsubmit="onSubmit()">
+      <fieldset class="block">
+        <legend>Personal information
+        </legend>
+        <label class="form-item" for="surname">
+          Surname:
+          <input type="text" name="surname"
+          id="surname">
+        </label>
+        <label class="form-item" for="name">
+          Name:
+          <input type="text" name="name"
+          id="name">
+        </label>
+        <label class="form-item" for="age">
+          How old are You?
+          <input type="number" id="age"
+          value="12" max="100">
+        </label>
+        <label class="form-item" for="date-of-birth">
+          Full date of birth:
+          <input type="date" name="date-of-birth"
+          id="date-of-birth">
+        </label>
+        <label class="form-item" for="terms">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms"
+          id="terms">
+        </label>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Registration
+        </legend>
+        <label class="form-item" for="email">
+          E-mail:
+          <input type="email" id="email"
+          placeholder="email@example.com" required>
+        </label>
+        <label class="form-item" for="password">
+          Password:
+          <input type="password" id="password"
```

Comment:
added empty line between tags
```suggestion
        <label class="form-item" for="email">
          E-mail:
          <input type="email" id="email"
          placeholder="email@example.com" required>
        </label>
        
        <label class="form-item" for="password">
          Password:
          <input type="password" id="password"
```